cloud_AWS/terraform/module/cloudexport.tf

```diff
@@ -7,22 +7,22 @@ terraform {
 }
 
 resource "kentik-cloudexport_item" "aws_export" {
-  # Create only when plan_id is set
-  count = var.plan_id == "" ? 0 : 1
+  # Create when bucket_name_list or plan_id is set
+  count = length(var.bucket_region_name) != 0 ? length(var.bucket_region_name) : var.plan_id == "" ? 0 : 1
```

Comment:
One of the comments mentions, that handling of plan_id == "" should be dropped as it is not needed anymore

Reply:
It is now using: 
`count = length(var.bucket_region_name) != 0 ? length(var.bucket_region_name) : var.create_cloudexport ? 1 : 0`
This is more explicit, still not perfect though.
Splitting the very module into submodules will require changes to other examples.